.gitignore

```diff
@@ -33,6 +33,9 @@ migrate_working_dir/
 .pub/
 /build/
 /packages/data/lib/utils/api_key_data.dart
+/packages/presentation/lib/utils/hidden_api_key.dart
```

Comment:
wow, we discussed to store api keys in yaml/json file, not in dart

Reply:
hear it for the first time

Reply:
changed